rpcs3/rpcs3qt/gui_settings.h

```diff
@@ -142,6 +142,7 @@ namespace gui
 	const gui_save ib_show_welcome = gui_save(main_window, "infoBoxEnabledWelcome",    true);
 	const gui_save ib_confirm_exit = gui_save(main_window, "confirmationBoxExitGame",  true);
 	const gui_save ib_confirm_boot = gui_save(main_window, "confirmationBoxBootGame",  true);
+	const gui_save check_upd_start = gui_save(main_window, "checkUpdateStart", true);
```

Comment:
Ib_ was meant as a shortcut for Infobox (or infodialog).
Feels weird to append a combobreaker here

Reply:
it's not an info box though, it's a check on boot for update, doesn't show or hide a dialog.

Reply:
Probably should put it in the mw section then

Reply:
Changed to m_check_upd_start and moved further down.